src/context/AuthContext.js

```diff
@@ -0,0 +1,18 @@
+import useAuthContext from './useAuthContext';
+
+// testing of the auth provider
+const initialState = {
+  email: 'valvic6@gmail.com',
```

Comment:
nu e prea ok sa faci push cu parole si clar nu e ok sa le pui ca initial state :)

Reply:
this was just for testing the new context.
removed. thanks.